prime-router/examples/weekly-report/reports-ingested.sql

```diff
@@ -0,0 +1,12 @@
+\echo List of all reports sent to SR  in the last week
```

Comment:
Just storing queries that I've put into the Metabase Dashboard, for reference.  There are several of these .sql files I put in examples/